.changeset/wide-donuts-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudflare/vite-plugin": minor
+---
+
+Add support for email triggers in `vite dev` and `vite preview`
```

Comment:
```suggestion
Add support for testing [Cron Triggers](https://developers.cloudflare.com/workers/configuration/cron-triggers/#test-cron-triggers-locally) and [Email Workers](https://developers.cloudflare.com/email-routing/email-workers/local-development/) in `vite dev` and `vite preview`.
```

Also, it does look like these docs should be updated to mention `vite dev` alongside `wrangler dev` if you have time.